### PR TITLE
Revise regularization notebooks and minor update for batch norm

### PR DESCRIPTION
**Pull request by astonzhang**

### CODE REVIEW

File: chapter_supervised-learning/reg-gluon.md [hunk 1]
```diff
@@ -43,16 +45,15 @@ data_iter_train = gluon.data.DataLoader(dataset_train, batch_size, shuffle=True)
 square_loss = gluon.loss.L2Loss()
 
 def test(net, X, y):
-    return square_loss(net(X), y).mean().asscalar()
+    return np.mean(square_loss(net(X), y).asnumpy())
```
Comment (mli) [inline chapter_supervised-learning/reg-gluon.md]: why need to use np.mean?
